CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Use the following format for additions: ` - VERSION: [feature/patch (if applicab
 - 1.1.19:
   - Ask before deleting a stash
   - Initial refactoring of click test using nightmare and mocha
+  - **Dropping support for node 4.x.**
+  - **Adding tests with node 7.x and the latest node. (currently 8.x)**
```

Comment:
Feels wrong to add changes to something that's already out :) Maybe either bump the version or do a line of `- Internal/No release` with the changes?